components/home/__tests__/homepage-start.test.tsx

```diff
@@ -0,0 +1,251 @@
+                  />
+                  <path
+                    clip-rule="evenodd"
+                    d="M5.887 13.282c.484 0 .877.393.877.877v.975c0 1.4 1.134 2.534 2.534 2.534h.488a.877.877 0 010 1.755h-.488a4.289 4.289 0 01-4.289-4.29v-.974c0-.484.393-.877.878-.877z"
+                    fill="currentColor"
+                    fill-rule="evenodd"
+                  />
+                </svg>
+              </div>
+              <a
+                class="text-gray-900 font-semibold text-xs mt-3 helper-link-cover"
+                href="/provinces?kebutuhan=Donor%20plasma"
+              >
+                Donor Plasma
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div
+          class="px-4 py-6"
+        >
+          <a
+            href="https://sembako.wargabantuwarga.com/"
+            rel="nofollow noopener noreferrer"
+            target="_blank"
+          >
+            <div
+              style="display: block; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
+            >
+              <div
+                style="display: block; box-sizing: border-box; padding-top: 35.97560975609756%;"
+              />
+              <img
+                alt="Ajukan bantuan sembako jika positif Covid-19 - Daftar Sekarang"
+                decoding="async"
+                sizes="100vw"
+                src="https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_3840,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg"
+                srcset="https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_640,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg 640w, https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_750,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg 750w, https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_828,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg 828w, https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_1080,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg 1080w, https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_1200,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg 1200w, https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_1920,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg 1920w, https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_2048,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg 2048w, https://res.cloudinary.com/wargabantuwarga/image/upload/c_scale,w_3840,q_90,cs_tinysrgb,f_auto/v1627206408/sembako-cta-v1_gpmaxg 3840w"
+                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
+              />
+            </div>
+          </a>
+        </div>
+      </section>
+    `);
+  });
+});
```

Comment:
If we want to keep this snapshot test once we migrate the content to Netlify CMS, we need to remember to mock the values, not relying on the `_content` directory which can be changed by our editors.